Data/ODBC/src/ODBCStatementImpl.cpp

```diff
@@ -225,6 +225,16 @@ void ODBCStatementImpl::bindImpl()
 	_pBinder->synchronize();
 }
 
+void ODBCStatementImpl::execDirectImpl(std::string query)
```

Comment:
I'm not sure, but it looks like you can use a constant reference here...

Reply:
It uses https://docs.microsoft.com/ru-ru/sql/odbc/reference/syntax/sqlexecdirect-function internally so nope

Reply:
ok =)